lib/go/test/flow_lockedtokens_test.go

```diff
@@ -2208,10 +2208,10 @@ func TestLockedTokensUnlockMultipleAccounts(t *testing.T) {
 			[][]byte{jsoncdc.MustEncode(cadence.Address(adminAddress))},
 		).(cadence.Dictionary).Pairs
 		assertEqual(t, 2, len(result))
-		assertEqual(t, nonLockedAddress.String(), result[1].Key.String()[2:])
-		assertEqual(t, CadenceUFix64("20000.0"), result[1].Value)
-		assertEqual(t, nonLocked2Address.String(), result[0].Key.String()[2:])
-		assertEqual(t, CadenceUFix64("30000.0"), result[0].Value)
+		assertEqual(t, nonLockedAddress.String(), result[0].Key.String()[2:])
```

Comment:
not sure why this happened, but the entries are now in reverse order.

Reply:
I know now that this is expected due to the change in how dictionaries are stored.